_docs/transaction-flow.md

```diff
@@ -1326,6 +1353,18 @@ For example, on an error you will receive a webhook like this:
 
 You can read more about problems during payments at our [error handling documentation]({{ "/docs/error-handling/" | prepend: site.baseurl }}).
 
+## State Reason Details
+
+Contains detailed information about error. `state_reason_details` contains fields: `code`, `category`, `messages`, `description`.
+
+There are six group of errors based on error codes:
```

Comment:
do we want to add the current list of specific error codes and corresponding descriptions under each of these generic error categories @sztupy ?

Reply:
Yes, let's make sure our doc is complete - otherwise how would the customers know what to prepare for?

Reply:
It won't be easy to present whole data in a table - on website. We have limited width (website), we cannot present it in the same way as in the spreadsheet. I will think about it, how to show more details

Reply:
mmm...maybe we could have a `Possible errors` link after the `description` where you're describing the `state_reason_details` which links to a table at the bottom of this page (or to a dedicated page, it might be better)...here's an idea of the table:


![Screenshot 2021-01-25 at 12 12 29](https://user-images.githubusercontent.com/40179292/105698785-a9508100-5f06-11eb-98f1-3094d25a4c5b.png)

Reply:
I was thinking how to present whole information in table and I have some proposition. For each code we could have something like this:
<img width="762" alt="Screenshot 2021-01-25 at 12 19 36" src="https://user-images.githubusercontent.com/945115/105699509-c20d6680-5f07-11eb-870c-47a0984703fb.png">
What do you think? @sztupy

Reply:
@emanuelegorga updated, could you recheck it?